projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -1448,6 +1012,21 @@
 			}
 		}
 
+		function loadBackgroundImage( slide ) {
+			if ( swiper && swiper.slides ) {
+				var currentSlide = swiper.slides[ swiper.activeIndex ];
+				currentSlide.style.backgroundImage = 'url(' + slide.attrs.mediumFile + ')';
+				return;
+			}
+			slide.el.style.backgroundImage = 'url(' + slide.attrs.mediumFile + ')';
```

Comment:
Assuming this is a pre-loader?

Maybe we can add some styles to the active element to hide it?

<img width="300" alt="Screen Shot 2021-06-23 at 3 58 48 pm" src="https://user-images.githubusercontent.com/6458278/123043628-eda36700-d43b-11eb-9b8b-852e180cae26.png">

E.g., 

```css
.swiper-slide-active {
	background-repeat: no-repeat;
    background-position-x: -100000vw;
}
```